### PR TITLE
Speedup CI. Reduce RabbitMQ network recovery interval during tests

### PR DESCRIPTION
**Pull request by remk**
Follow the work done in  #3190

### CODE REVIEW

File: mailbox/api/src/test/java/org/apache/james/mailbox/events/GroupContract.java [hunk 1]
```diff
@@ -96,7 +96,7 @@ public void event(Event event) throws Exception {
             IntStream.range(0, eventCount)
                 .forEach(i -> eventBus().dispatch(EVENT, NO_KEYS).block());
 
-            WAIT_CONDITION.atMost(org.awaitility.Duration.TEN_MINUTES).untilAsserted(() -> assertThat(finishedExecutions.get()).isEqualTo(eventCount));
+            waitCondition(getSpeedProfile()).atMost(org.awaitility.Duration.TEN_MINUTES).untilAsserted(() -> assertThat(finishedExecutions.get()).isEqualTo(eventCount));
```
Comment (chibenwa) [inline mailbox/api/src/test/java/org/apache/james/mailbox/events/GroupContract.java]: Why don't the speed profile returns it's wait condition?

**Comment by Arsnael** [comment]
```
[e1094fcd9fd3d6bf21c8b60927770e3540f64ffc] [ERROR]   RabbitMQEventDeadLettersIntegrationTest.multipleFailedEventsShouldBeCorrectlyProcessedByListenerAfterSuccessfulGroupRedelivery:484 » ConditionTimeout
```

**Comment by chibenwa** [comment]
test this please

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**